projects/lib-shared-modules/src/lib/components/header/header.component.html

```diff
@@ -1 +1,3 @@
-<p>header works updated!!!</p>
+<header>
+    <a><span defaultHref="" *ngIf="backButton" (click)="backArrowButton()">&#8592;</span>Creation Portal</a>
```

Comment:
Show back button as per condition from parent by getting data from parent component whether to show or not.